README.md

```diff
@@ -22,13 +22,13 @@ $ pip3 install yark
 Once you've installed Yark, think of a name for your archive and copy the target's url:
 
 ```shell
-$ yark new owez https://www.youtube.com/channel/UCSMdm6bUYIBN0KfS2CVuEPA
+$ yark new my-archive https://www.youtube.com/channel/UCSMdm6bUYIBN0KfS2CVuEPA
```

Comment:
I think `foobar` would be better because I don't want to put any naming conventions for `hello-there` or `hello_there` or `HelloThere` for archive names at the moment